Tensile/Tests/conftest.py

```diff
@@ -157,3 +157,20 @@ def __exit__(self, exc_type, exc_value, traceback):
 
     return gpUpdater
 
+@pytest.fixture
+def initGlobalParametersForTCL():
+    from Tensile import Common
+    from Tensile.TensileCreateLib.ParseArguments import parseArguments    
```

Comment:
Move these to top-level imports?

Reply:
I was keeping with convention for the file. Do you have a preference?

Reply:
Top-level as the preference makes sense to me, ensures import errors are thrown early, and makes it easy to see at a glance how complex the file is. However, as a new function, I don't see an issue keeping it there until it becomes more permanent fixture.